kamon-kafka-clients/src/main/java/kamon/kafka/client/instrumentation/advisor/Advisors.java

```diff
@@ -1,6 +1,5 @@
-/*
- * =========================================================================================
- * Copyright © 2013-2019 the kamon project <http://kamon.io/>
+/* =========================================================================================
```

Comment:
Please, split this file into two: one for the producer advice and another for the consumer advice.

Reply:
Done